oceanprefilter/oceanprefilter.go

```diff
@@ -179,22 +169,40 @@ func (pf *prefilter) Reconfigure(ctx context.Context, deps resource.Dependencies
 		theZone := image.Rectangle{image.Point{er[0], er[1]}, image.Point{er[2], er[3]}}
 		rc.excludedZone = &theZone
 	}
+	ensemble, err := xgb.LoadXGBoostFromJSONBytes(modelbytes,
+		"", 2, 8, &activation.Softmax{}) // wrong activation logistic is 1 class only
```

Comment:
you don't have to put the comment in here -- I don't think someone working on this after you will have known about the error with the Logistic activation function